source/features/follow-file-renames.tsx

```diff
@@ -8,50 +8,69 @@ import * as api from '../github-helpers/api';
 import GitHubURL from '../github-helpers/github-url';
 
 interface File {
-	previous_filename: string;
+	previous_filename: string | undefined;
 	filename: string;
 	status: string;
+	blob_url: string;
 }
-// eslint-disable-next-line import/prefer-default-export
-export async function getCommitInfo(oid: string): Promise<AnyObject> {
+interface CommitInfo {
+	parentSha: string | undefined;
+	date: Date;
+	url: string;
+}
+export interface FileChanges {
+	commit: CommitInfo;
+	file: File;
+}
+
+export async function getChangesToFileInCommit(sha: string, filePath: string): Promise<FileChanges | void> {
 	// API v4 doesn't support it: https://github.community/t/what-is-the-corresponding-object-in-graphql-api-v4-for-patch-which-is-available-in-rest-api-v3/13590
-	return api.v3(`commits/${oid}`);
+	const commit = await api.v3(`commits/${sha}`);
+	const commitInfo: CommitInfo = {
+		parentSha: commit.parents[0],
+		date: commit.commit.committer.date,
+		url: commit.html_url,
+	};
+	for (const fileInfo of commit.files as File[]) {
+		if ([fileInfo.filename, fileInfo.previous_filename].includes(filePath)) {
+			return {
+				commit: commitInfo,
+				file: fileInfo,
```

Comment:
I meant this
```suggestion
				commit: {
		 		    parentSha: commit.parents[0],
		 		    date: commit.commit.committer.date,
		 		    url: commit.html_url,
		 	    },
				file: fileInfo,
```

Reply:
Changed to the above, removed the `CommitInfo` interface